packages/ses/src/scope-handler.js

```diff
@@ -136,17 +134,7 @@ export const createScopeHandler = (
     // ReferenceError for such assignments)
 
     has(_shadow, prop) {
-      // unsafeGlobal: hide all properties of the current global
-      // at the expense of 'typeof' being wrong for those properties. For
-      // example, in the browser, evaluating 'document = 3', will add
-      // a property to globalObject instead of throwing a ReferenceError.
-      return (
-        sloppyGlobalsMode ||
-        prop === 'eval' ||
-        prop in localObject ||
-        prop in globalObject ||
-        prop in globalThis
-      );
+      return typeof prop === 'string';
```

Comment:
I would feel more comfortable here if we simply returned `true`. I know symbol properties are not possible for scope lookup, and I'm not aware of environments defining any on the global object, but since the scope proxy leaks, there is a risk that code with a handle on the proxy object could probe symbols of the start compartment's global (if any).